### PR TITLE
Version 20.2.0 → 21.0.0

### PR DESCRIPTION
**Pull request by Gabriel439**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,58 @@ file.
...
+  * [Add parser failure test for whitespace before `with`](https://github.com/dhall-lang/dhall-lang/pull/1184)
+  * [Fix Prelude test files to use `.dhall` suffix](https://github.com/dhall-lang/dhall-lang/pull/1186)
+  * [Test changes for `dhall-purescript`](https://github.com/dhall-lang/dhall-lang/pull/1190)
+
+New features:
+
+* [Add Prelude support for NonEmpty lists](https://github.com/dhall-lang/dhall-lang/pull/1177)
+
+
+
```
Comment (TristanCacqueray) [inline CHANGELOG.md:59]: nit: the previous versions are only separated with one empty line.

**Review by sjakobi — APPROVED** [review]
